src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -113,7 +113,7 @@
                     sfwdt.setAttribute('class', 'sf-toolbar sf-error-toolbar');
                 }
             },
-            { maxTries: 5 }
+            { maxTries: 5, timeoutDelay: 1000 }
```

Comment:
`timeoutDelay` here, while it's `retryDelay` the other way around? :)

Reply:
just fixed it this minute. stupid mistake